### DISCUSSION TITLE
How to support 7.8.0

### DISCUSSION THREAD

**Discussion by lvinkim** [fbaligand/kibana-enhanced-table#159] [General]
Only 7.8.1 can choose . But got error :

Plugin installation was unsuccessful due to error "Plugin enhanced-table [7.8.1] is incompatible with Kibana [7.8.0]"

**Comment by fbaligand** [comment]
Hi,

Kibana 7.8.0 is not supported because there is a blocking issue with Kibana 7.8.0 (#116), that has been fixed in Kibana 7.8.1.
So there is no release of enhanced Table for Kibana 7.8.0
I invite you to upgrade to Kibana 7.8.1. You can keep elasticsearch 7.8.0
